src/main/scala/formal/FormalUtils.scala

```diff
@@ -46,12 +46,13 @@ object Property {
 
   def apply(dir: MonitorDirection, cond: Bool, message: String,
             prop_type: PropertyClass=PropertyClass.Default,
-            idx: String = "")(implicit sourceInfo: SourceInfo): Unit = {
+            idx: String = "",
```

Comment:
We should take the chance to actually git rid of this.  It used to prevent it from generating the same label, but firrtl have  a built in mechanism for that now.

Reply:
As discussed, we should make this change in a separate PR.